packages/lsp-extension/src/renderer.tsx

```diff
@@ -218,6 +218,7 @@ function BuildSettingForm(props: ISettingFormProps): JSX.Element {
                   <div className="jp-inputFieldWrapper jp-FormGroup-contentItem">
                     <input
                       className="form-control"
+                      aria-label={props.trans.__('Server name')}
```

Comment:
Here too, I think it would be better to connect this input with the "Server name" above. Will probably need to change the `<h3>` to a `<label>`